modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_26_0/ObjectFieldSettingUpgradeProcess.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.upgrade.v3_26_0;
+
+import com.liferay.object.constants.ObjectFieldSettingConstants;
+import com.liferay.object.constants.ObjectRelationshipConstants;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.StringBundler;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.uuid.PortalUUID;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Timestamp;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class ObjectFieldSettingUpgradeProcess extends UpgradeProcess {
+
+	public ObjectFieldSettingUpgradeProcess(PortalUUID portalUUID) {
+		_portalUUID = portalUUID;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement selectPreparedStatement =
+				connection.prepareStatement(
+					StringBundler.concat(
+						"select ObjectField.objectFieldId, ",
+						"ObjectField.companyId, ObjectField.userId, ",
+						"ObjectField.userName, ObjectField.name from ",
+						"ObjectField where ObjectField.relationshipType = '",
+						ObjectRelationshipConstants.TYPE_ONE_TO_MANY, "'"));
```

Comment:
I'm not sure if we should we use the `relationshipType` or the `businessType` 😕

Reply:
I think by relationshipType is more accurate because if we choose to go with businessType, we would have to check which type of relationship (2 steps) :thinking: